parameter/FixedPointParameterType.cpp

```diff
@@ -269,8 +271,8 @@ size_t CFixedPointParameterType::getUtilSizeInBits() const
 // Compute the range for the type (minimum and maximum values)
 void CFixedPointParameterType::getRange(double& dMin, double& dMax) const
 {
-    dMax = (double)((1UL << (_uiIntegral + _uiFractional)) - 1) / (1UL << _uiFractional);
-    dMin = -(double)(1UL << (_uiIntegral + _uiFractional)) / (1UL << _uiFractional);
+    dMax = (double)((1UL << (_uiIntegral + _uiFractional)) - 1) / double(1UL << _uiFractional);
+    dMin = -(double)(1UL << (_uiIntegral + _uiFractional)) / double(1UL << _uiFractional);
```

Comment:
Maybe use `double{...}` to highlight the fact that their is no precision loose ?